SantaFinder.ts

```diff
@@ -1,110 +1,110 @@
-import { Santa } from "./types"
+import { Santa, SantaNode } from "./types"
 
 export class SantaFinder {
-    santas: { [key: string]: Santa }
-    availablePresentees: { [key: string]: Santa }
-    possibleCycles: Array<Array<string>> = []
+    santas: Array<Santa>
 
-    constructor(players: Array<Santa>) {
-        this.santas = {}
-        this.availablePresentees = {}
+    availableSantas: Array<string>
+    availablePresentees: Array<string>
 
-        players.forEach(player => {
-            this.santas[player.id] = player
-            this.availablePresentees[player.id] = player
-        })
+    constructor(players: Array<Santa>) {
+        this.santas = players
+        this.availableSantas = players.map((player) => { return player.id })
```

Comment:
```suggestion
        this.availableSantas = players.map((player) => player.id)
```

Wenn du nur einen Wert returnst kannst du hier die Short-Form nutzen :)